react-challenge.html

```diff
@@ -35,24 +35,32 @@
 
         handleMoodOptions = () => {
           let randomIndex = Math.floor(Math.random() * this.state.moods.length )
-          this.setState({ currentMood: moods[randomIndex] })
+          this.setState({ currentMood: this.state.moods[randomIndex] })
```

Comment:
Yea, Great job here getting it working.   Generic advice: With React and future bug fix challenges try to practice error logging as a source of information to get a workflow down.